<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="idInputLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>UPC:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="upcInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Search</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="looseButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Loose</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="completeButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete in &amp;Box</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="sealedButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>84</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;New</string>
    </property>
   </widget>
   <widget class="QLabel" name="conditionLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Condition:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Game:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="consoleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Console:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="priceLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Price:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="gameOutput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="consoleOutput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceOutput">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>280</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Add To Collection</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>83</x>
      <y>330</y>
      <width>84</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="looseTotalLabel">
       <property name="text">
        <string>Loose Total: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="completeTotalLabel">
       <property name="text">
        <string>Complete Total: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sealedTotalLabel">
       <property name="text">
        <string>Sealed Total: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="totalValueLabel">
       <property name="text">
        <string>Collection Value: </string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>110</width>
      <height>117</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="looseTotalOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="completeTotalOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sealedTotalOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="totalValueOutput">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clearCollectionButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Clear Collection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearFormButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>1</number>
    </property>
    <property name="text">
     <string>Cl&amp;ear Results</string>
    </property>
    <property name="shortcut">
     <string>Alt+E</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>377</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame_Value_Calculator">
    <property name="title">
     <string>Game Value Calculator</string>
    </property>
   </widget>
   <addaction name="menuGame_Value_Calculator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>getPrice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>166</x>
     <y>310</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearCollectionButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearCollection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearFormButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>clearForm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>getPrice()</slot>
  <slot>addItem()</slot>
  <slot>clearCollection()</slot>
  <slot>clearForm()</slot>
 </slots>
</ui>
